js/apps/system/_admin/aardvark/APP/frontend/js/collections/arangoCollections.js

```diff
@@ -167,7 +167,11 @@
         data.shardKeys = object.shardBy;
       }
       if (object.replicationFactor) {
-        data.replicationFactor = JSON.parse(object.replicationFactor);
+        if (typeof object.replicationFactor === 'number' && frontendConfig.isEnterprise) {
+          data.replicationFactor = JSON.parse(object.replicationFactor);
+        } else {
+          data.replicationFactor = object.replicationFactor;
+        }
       }
```

Comment:
What is this "if" supposed to do? 
If the input value is already a number (typeof ... === "number") then the JSON.parse will not change anything.
What happens if the input is a string that contains only numeric digits, e.g. "123"? Should this then be converted into a number? The above code is probably not addressing this.